android/src/main/java/cl/json/social/InstagramStoriesShare.java

```diff
@@ -0,0 +1,46 @@
+package cl.json.social;
+
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import java.io.File;
+import android.os.Environment;
+import android.net.Uri;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReadableMap;
+
+/**
+ * Created by Vladimir Stalmakov on 01-06-20.
+ */
+public class InstagramStoriesShare extends SingleShareIntent {
+
+    private static final String PACKAGE = "com.instagram.android";
+    private static final String PLAY_STORE_LINK = "market://details?id=com.instagram.android";
```

Comment:
Why we don't use this url "https://play.google.com/store/apps/details?id=com.instagram.android" instead?

Reply:
I used a link like from InstagramShare.java. Fixed

Reply:
I get it, maybe we can change this later. I think redirecting to Google Play is most common now.